WpfMath/TexFormulaParser.cs

```diff
@@ -140,7 +142,7 @@ private static bool IsWhiteSpace(char ch)
         public TexFormulaParser()
         {
             if (!isInitialized)
-                throw new InvalidOperationException("Parser has not yet been initialized.");
+                Initialize();

```

Comment:
I think that we should put that code into a static constructor. But, please, be careful and remember the rules for the static ctor execution.

Current code is not thread-safe, and I'd like it to be safe because I want to run WPF-Math on the server side.